spec/overview.html

```diff
@@ -20,7 +20,7 @@ <h1>Internationalization, Localization, and Globalization</h1>
     <h1>API Overview</h1>
 
     <p>
-      The ECMAScript 2023 Internationalization API Specification is designed to complement the ECMAScript 2023 Language Specification by providing key language-sensitive functionality. The API can be added to an implementation of the ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, or successor).
+      The ECMAScript 2023 Internationalization API Specification is designed to complement the ECMAScript 2023 Language Specification by providing key language-sensitive functionality. The API can be added to an implementation of the ECMAScript 2023 Language Specification (ECMA-262 14<sup>th</sup> Edition, or successor) in whole or in part. Excepting interaction with language values introduced by this specification that are observable to ECMAScript code (such as the value of a [[FallbackSymbol]] internal slot and the set of values transitively reachable from %Intl% by <emu-xref href="#sec-property-accessors">property access</emu-xref>), nothing required by this specification is incompatible with behaviour permitted by ECMA-262 to conforming implementations and nothing prohibited by this specification is permitted by ECMA-262 to conforming implementations.
```

Comment:
why "or in part"? why would we want something to implement part of Intl?

Reply:
It's not necessarily something we _want_, but it should be allowed for an implementation to adopt ECMA-402 incrementally (e.g., `Intl.DisplayNames` before `Intl.PluralRules`). And that is in fact the case even if not mentioned here, because ECMA-262 does not prohibit any implementation from adding an `Intl` property to the global object independent of ECMA-402.

Reply:
I'll remove "in whole or in part" if you feel strongly about it, but I don't see text in either specification that would cause an implementation of ECMA-262 providing an `Intl.DisplayNames` built-in without claiming ECMA-402 support from being considered nonconforming with the former or in conflict with the latter. The clause is intended as recognition of the reality that interfaces specified in ECMA-402 can be incrementally added to a conforming implementation of ECMA-262.

Reply:
They certainly can, but doing so would not be in compliance with 402 until the implementation was complete.

Reply:
Are we saying the same thing? This is not a statement about ECMA-402 conformance at all, but rather about its layering relationship with ECMA-262—specifically, that each part of it is independently compatible with ECMA-262.

Reply:
Hmm, maybe we’re not. My concern is that nobody should be able to claim conformance to 402 with a partial implementation.

If this line doesn’t create that risk, then feel free to resolve it.